mindocr/postprocess/det_postprocess.py

```diff
@@ -22,15 +22,15 @@ def __init__(self, binary_thresh=0.3, box_thresh=0.7, max_candidates=1000, expan
         self._name = pred_name
         self._names = {'binary': 0, 'thresh': 1, 'thresh_binary': 2}
 
-    def __call__(self, pred, **kwargs):
+    def __call__(self, pred: Union[Tensor, Tuple[Tensor], np.ndarray], img_shape: Tuple[int, ...], **kwargs) -> dict:
```

Comment:
1. we need to leave it an option to control whether resize back to the original shape in postprocessing, because the evaluation performance will differ slightly in these two cases. The overall result of detection then recognition will differ more. 
2. Knowing the image shape is not enough to map the points back to the original space correctly. If we apply padding in resizing, e.g. scale input hw in shape [500, 500] by 1.2 then pad to [736, 1280] in inference, the predicted polygon points should be mapped by pred_x / 1.2, instead of pred_x / (1280/500).
3. Add illustration for img_shape in comments.

Reply:
Done